hello-strongbox-npm/npmrc.template

```diff
@@ -0,0 +1,5 @@
+registry=http://localhost:48080/storages/storage-npm-common/npm-releases
+username=user
+_password=password
```

Comment:
Have you, by any chance tried it with `_auth`? This is the other way to define credentials (the format of the value is a Base64-encoded representation of the `user:pass` combination. If you have, could you add a commented line with the `_auth=theEncodedCredentials`, just so that people know...? Thanks! :)

Reply:
done